form.html

```diff
@@ -40,16 +42,17 @@ <h1>What's the occassion?</h1>
             <section id="generateActivity">
                 <h1>Don't know?<br> Let us choose an activity for you!</h1>
                 <!-- Link onclick event when function is created in js-->
-                <button id="boredAPI" onclick="getActivity()">Generate Activity</button>
+                <button class="startButton" onclick="getActivity()">Generate Activity</button>
```

Comment:
keep the button ID. so <button id ="boredAPI" class="startButton".....> Same thing for the other